streams/src/main/java/org/apache/kafka/streams/kstream/internals/InternalStreamsBuilder.java

```diff
@@ -34,36 +34,35 @@
 public class InternalStreamsBuilder implements InternalNameProvider {
 
     final InternalTopologyBuilder internalTopologyBuilder;
+    private final StreamsTopologyGraph topologyGraph;
```

Comment:
Do we still need InternalTopologyBuilder after we have intercepted all the calls?

Reply:
I have merged functionality from `StreamsTopologyGraph` into `InternalStreamsBuilder` and removed `StreamsTopologyGraph`